templates/profile.html

```diff
@@ -142,29 +142,35 @@
     }
 
     .avatar-list {
-      display: none; /* Hidden by default */
-      position: absolute;
-      background: #fff;
-      border: 1px solid #ccc;
-      border-radius: 6px;
-      padding: 10px;
-      box-shadow: 0 2px 5px rgba(0,0,0,0.2);
-      z-index: 999;
-    }
-
-    .avatar-list img {
-      width: 50px;
-      height: 50px;
-      border-radius: 50%;
-      margin: 5px;
-      border: 2px solid transparent;
-      cursor: pointer;
-      transition: border 0.3s;
-    }
-
-    .avatar-list img:hover {
-      border-color: #c28adc;
-    }
+  display: none; /* Hidden by default */
+  position: absolute;
+  background: #fff;
+  border: 1px solid #ccc;
+  border-radius: 6px;
+  padding: 10px;
+  box-shadow: 0 2px 5px rgba(0,0,0,0.2);
+  z-index: 999;
+
+  /* Grid layout for 5 rows and 2 columns */
+  display: grid;
+  grid-template-columns: repeat(2, 1fr); /* 2 columns */
+  gap: 10px; /* Space between items */
+  width: 120px; /* Adjust width for the grid */
+}
+
+.avatar-list img {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  margin: 0; /* Remove margin to fit grid properly */
+  border: 2px solid transparent;
+  cursor: pointer;
+  transition: border 0.3s;
+}
+
+.avatar-list img:hover {
+  border-color: #c28adc;
+}
```

Comment:
II think it would be nice if we have all the styling in the css file within the static folder to keep it consistent with the rest of the code!